src/views/MouseDiffExp/store.js

```diff
@@ -31,6 +31,8 @@ export default new Vuex.Store({
         tissueToQuery: "",
         geneToQuery: "",
         selectedAncestry: "",
+        loadingGene: true,
+        loadingExpression: true,
```

Comment:
These should default to _false_. If these states are accessed by other outside components, they will be read as "loading" at startup, even though nothing is.

Reply:
Fixed, thank you.